server/models/HistoryEra.js

```diff
@@ -45,6 +47,18 @@ class HistoryEra {
     this.startAutoCleanup(cleanupInterval, this.db);
   }
 
+  loadDatabase(userId, dbName) {
+    let dbPath = `${config.dbPath}${userId}/`;
```

Comment:
This should be `const`, but let's just concatenate the string all at once in the `filename` key assignment.